src/loaders/api/api.js

```diff
@@ -81,12 +80,10 @@ export function setAPI (agentIdentifier, forceDrain) {
     const currentInfo = getInfo(agentIdentifier)
     if (value === null) {
       delete currentInfo.jsAttributes[key]
-      if (isBrowserScope) removeFromBrowserStorage(key, CUSTOM_ATTR_GROUP) // addToBrowserStorage flag isn't needed to unset keys from storage
     } else {
       setInfo(agentIdentifier, { ...currentInfo, jsAttributes: { ...currentInfo.jsAttributes, [key]: value } })
-      if (isBrowserScope && addToBrowserStorage) putInBrowserStorage(key, value, CUSTOM_ATTR_GROUP)
     }
-    return apiCall(prefix, apiName, true)()
+    return apiCall(prefix, apiName, true, !!addToBrowserStorage || (value === null && 'session'))(key, value)
```

Comment:
```suggestion
    return apiCall(prefix, apiName, true, !!addToBrowserStorage || value === null)(key, value)
```

Reply:
This is the second weird selector. I wonder if the lint tools are doing something weird

Reply:
will fix